packages/headless/package.json

```diff
@@ -28,7 +28,7 @@
     "build:prod": "npm run typedefinitions && rollup -c --environment BUILD:production",
     "typedefinitions": "tsc -p src/tsconfig.build.json -d --emitDeclarationOnly --declarationDir dist/definitions",
     "clean": "rimraf -f -r dist/*",
-    "test": "./node_modules/jest/bin/jest.js",
+    "test": "jest",
```

Comment:
I'm not sure why, but the path [was specified](https://github.com/coveo/ui-kit/pull/1170/files#diff-b49fb4b1d19b4b6568e5ff36ec318c7410ab21ce4405d21930510332d64c3905R30) for this script a few months ago. However, it causes an intermittent error on my machine which requires me to reinstall jest to resolve:

![npm-test_error](https://user-images.githubusercontent.com/5565841/143907573-7aea7bfe-14da-4707-8705-f4c936950b2d.PNG)


If there's a good reason for it, then I can adjust the path to `./node_modules/.bin/jest` (using `.bin` instead of `jest` dir).

Reply:
For node upgrade, to run properly on Jenkins.

Right now the build has failed at that step : 

```
> @coveo/auth@1.1.1 test
> jest

lerna ERR! npm run test exited 127 in '@coveo/headless'
lerna ERR! npm run test stdout:

> @coveo/headless@1.36.0 test
> jest


lerna ERR! npm run test stderr:
sh: 1: jest: not found
```

Reply:
Also, just for info, how do you debug tests in your machine ? 

Are you not using vscode Jest plugin ?

Reply:
Hmm, I wonder why it works in `@coveo/auth` but not `@coveo/headless`.

For debugging an individual test, I use the jest plugin. For running all tests, I run `npm test`, and when writing new tests, I use `npm run test:watch`.

Reply:
I updated the `jest` dependencies. I fixed certain tests according to the latest types, they should now run on jenkins using just the `jest` command.